src/sensor/sensor.py

```diff
@@ -0,0 +1,23 @@
+from doctest import OutputChecker
+from random import random
+from Utils.status import Status
+
+
+class sensor:
```

Comment:
Refactor Sensor class name to UpperCamelCase
```suggestion
class Sensor:
```